src/peft/tuners/lycoris_utils.py

```diff
@@ -79,7 +79,8 @@ def __init__(self, base_layer: nn.Module) -> None:
 
     @property
     @abstractmethod
-    def _available_adapters(self) -> set[str]: ...
+    def _available_adapters(self) -> set[str]:
+        ...
```

Comment:
`make style` did this. Should I revert?

Reply:
Not necessary. This change should already be on latest main, so you could merge with/rebase on main to remove this diff.

Reply:
I don't think that's the case. I rebased on main before I started work on the changes earlier.

Reply:
Ah yes, then it's most likely because you have an older ruff version installed. Try v0.4.10.

Reply:
That was it!